content/ops/security-ir.md

```diff
@@ -0,0 +1,180 @@
+# cloud.gov Incident Response process
+
+This document outlines cloud.gov's process for responding to security incidents. It outlines roles and responsibilities during and after incidents, and lays out the steps we'll take to resolve them.
+
+## Response process
+
+### Initiate
+
+An incident begins when someone (the *reporter*) becomes aware of a potential incident. We define "incident" broadly, following [NIST SP 800-61](http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-61r2.pdf), as "a violation or imminent threat of violation of computer security policies, acceptable use policies, or standard security practices" (6). This is a deliberately broad definition, designed to encompass any scenario that might threaten the security of cloud.gov.
+
+The reporter should contact [FIXME: initial contact?] to become the response process. They should escalate [FIXME: how?] until they receive acknowledgment of their report.
```

Comment:
In order to comply with GSA, the very first thing they should *always* do is email gsa-ir@gsa.gov _and_ itservicedesk@gsa.gov until further notice.